testsuite/python/correlation.py

```diff
@@ -52,12 +52,12 @@ def test_square_distance_componentwise(self):
 
         obs = espressomd.observables.ParticlePositions(ids=(0,))
         acc = espressomd.accumulators.Correlator(
-            obs1=obs, tau_lin=10, tau_max=10.0, delta_N=1,
+            obs1=obs, tau_lin=10, tau_max=2, delta_N=1,
```

Comment:
during the rebase, you accidentally discarded the merge conflict resolutions; the `tau_max=2` change must be applied to the new test cases at lines 83, 108 and 133 (but not 189, because this test doesn't run the integrator and the custom `tau_max` value there is important)